tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml

```diff
@@ -87,6 +87,24 @@ exon sequences and transcript structures.
 is used as an input for ``ribotaper part 2``.
 *Annotation_path*
 is used as an input for ``ribotaper part 3``.
+
+Important notes
+----------------
+
+- To install the galaxy tool, please enable conda installation in **galaxy.ini**
+
+- Please add the following line to **datatypes_conf.xml** to enable the data type, **compressed_archive**,
+
+    <datatype extension="compressed_archive" type="galaxy.datatypes.binary:CompressedArchive" subclass="True" display_in_upload="True"/>
```

Comment:
@bgruening I tested tar vs zip, tar is a better choice. 

do u want me to make a new pull request or you could change zip to tar?

In addition, I don't find this version in the main toolsheld where has an older version.